flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mongodb-cdc/src/test/java/org/apache/flink/cdc/connectors/mongodb/source/MongoDBFullChangelogITCase.java

```diff
@@ -648,6 +648,7 @@ private void testMongoDBParallelSource(
                                 + " name STRING,"
                                 + " address STRING,"
                                 + " phone_number STRING,"
+                                + " op_type STRING METADATA FROM 'op_type' VIRTUAL,"
```

Comment:
Thanks @qg-lin for this improvement.
How about add this case into `MongoDBConnectorITCase#testMetadataColumns`